misoc/integration/soc_core.py

```diff
@@ -224,7 +226,7 @@ def build(self, *args, **kwargs):
 
 def soc_core_args(parser):
     parser.add_argument("--cpu-type", default=None,
-                        help="select CPU: lm32, or1k, vexriscv")
+                        help="select CPU: lm32, or1k, vexriscv, riscv32g")
```

Comment:
That's not very consistent naming... How about vexriscv-g?

Reply:
2198485